cumulusci/core/keychain/base_project_keychain.py

```diff
@@ -53,16 +118,12 @@ def _convert_connected_app(self):
                 "client_secret": connected_app.client_secret,
             }
         )
-        self.set_service("connected_app", ca_config)
+        # We're using an obnoxious alias to see who's using this
+        self.set_service("connected_app", "please_contact_sfdo_releng", ca_config)
```

Comment:
@davisagli - Do you want me to just rip out the `_convert_connected_app()` function (and all references to it) in BaseProjectKeychain? (This would make me quite happy :) )

Reply:
Yeah, I think we should go ahead and rip out everything that's related to connected apps as a thing that is separate from services, and just be sure to call it out in the release notes. This can be part of this PR or a new story for another sprint.